Create is-innersource-right.md

rrrutledge opened the pull request:
View the [rendered markdown](https://github.com/rrrutledge/innersource-program/blob/rrrutledge-patch-3/articles/is-innersource-right.md).

rrrutledge committed:
Create is-innersource-right.md

rrrutledge committed:
Update is-innersource-right.md

spier commented on articles/is-innersource-right.md:
```diff
@@ -0,0 +1,33 @@
+
+## Is it Company Specific?
+
+InnerSource projects make sense when the project is specific to the company or gives the company a strategic business advantage. Other collaborative projects should be run as open source so as to increase the contribution pool and impact.
```
```suggestion
InnerSource projects make sense when the project is specific to the company or gives the company a strategic business advantage. Other collaborative projects should be run as open source to increase the contribution pool and impact.
```

simpler language

spier commented on articles/is-innersource-right.md:
```diff
@@ -0,0 +1,33 @@
+
+## Summary
+
+If your project is company-specific *and* contributions will come **and** you will support those contributions, then InnerSource is right for your project.
```
Maybe the 3 questions in the same order as they appear in the document above?

To do that, one could move the "Is it Company Specific?" to the top.
Would make sense to me as that should be a simple test/question that needs to be answered with "No" first, before the readers proceed to the next questions.

spier replied:
```suggestion
If your project is company-specific **and** contributions will come **and** you will support those contributions, then InnerSource is right for your project.
```

Did you want to use the same formatting for **and**?

spier commented on articles/is-innersource-right.md:
```diff
@@ -0,0 +1,33 @@
+
+## Will Contributions Come?
+
+An InnerSource approach only makes sense if contributions are expected from the project's users. You can expect contributions to come if you see or anticipate noticeable amounts of energy directed at your project space by its users. Some examples:
```
What does "project space" mean? Why not just "project"?

rrrutledge replied:
If users are home-rolling workarounds that are in your project space (but not necessarily your exact project) then that is also a good reason to run your project as InnerSource.

spier commented on articles/is-innersource-right.md:
```diff
@@ -0,0 +1,33 @@
+
+-   Understanding users' scenarios and suggesting what contributions in your project could help them to meet those scenarios.
+-   Inviting users to make the contributions that they need and following up with them to ensure that they do them.
+-   Maintaining an CONTRIBUTING.md document that contains everything an engineer needs to know to contribute to the project.
```
The is an opportunity link to even more patterns here, if that is desirable.
e.g. https://patterns.innersourcecommons.org/p/base-documentation#contributing.md

spier reviewed (approved):
I really like the document already in its current form.

I left some inline comments for areas that could be improved.

rrrutledge commented:
Thank your for this review, @spier ❗   I moved this content to the [InnerSource Learning Path](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/481) and incorporated your suggestions!
